recipes/circularbuffer/all/conanfile.py

```diff
@@ -11,24 +17,32 @@ class CircularBufferConan(ConanFile):
     homepage = "https://github.com/rlogiacco/CircularBuffer"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    def package_id(self):
-        self.info.header_only()
-
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+        get(self, **self.conan_data["sources"][self.version],
+            destination=self.source_folder, strip_root=True)
+
+    def layout(self):
+        basic_layout(self)
```

Comment:
Shouldn't you use a cmake_layout here ?

Reply:
I don't know. Fresh from the 2.0 workshop. Please tell me 

It works on my machine ...

Reply:
No, `rlogiacco/CircularBuffer` is a header-only so this template applies https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/header_only/all/conanfile.py which uses `basic_layout`.

(It's missing the `src_root="sdk"` param though:
```suggestion
        basic_layout(self, src_folder="src")
```

Reply:
+1 for the response above ;)